.travis.yml

```diff
@@ -22,14 +36,10 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower
-  - bower --version
-  - npm install phantomjs-prebuilt
-  - node_modules/phantomjs-prebuilt/bin/phantomjs --version
-
-install:
-  - npm install
-  - bower install
+  - npm install -g npm@4
+  - npm --version
```

Comment:
CI is failing due to needing Bower install still, we should either bring that back or port the few Bower dependencies to use the NPM equivalents. Feel free to whichever you think is easier.

Reply:
CI now passing. Thanks for the suggestion.